zerver/tests/test_signup.py

```diff
@@ -1857,6 +1854,16 @@ def test_create_education_demo_organization_welcome_bot_direct_message(self) ->
         self.assertIn("using Zulip for a class guide", welcome_msg.content)
         self.assertIn("demo organization", welcome_msg.content)
 
+        # Organization owner has expected defaults for demo organization creation.
+        owner = realm.get_human_owner_users().last()
```

Comment:
I think there is a possibility that this can return a different user, so we should use a different approach to test the created user.